src/main/java/fmartin1/common/NamedAPIContainer.java

```diff
@@ -0,0 +1,7 @@
+package fmartin1.common;
+
+import java.util.List;
+
+public interface NamedAPIContainer {
```

Comment:
Add a comment to explain the value of this class

Reply:
This class has been removed